File: java/form with checkbox/MainActivity.java
package com.gradproject.formwithcheckbox;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    EditText getName;
    TextView setName;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        getName = findViewById(R.id.editTextTextPersonName);
        setName = findViewById(R.id.textView2);
    }
    public void confirm(View view){
        String name = getName.getText().toString();
        setName.setText("Welcome "+name);

    }
}

File: Kotlin/data entry ionic/Complete app/app/src/main/java/com/badawy5/app5/MainActivity.kt
package com.badawy5.app5

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*
import java.util.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        button1.setOnClickListener(
            {
                val Name: String = editTextTextPersonName.text.toString()

                textView2.text = "Welcome  " + Name


            }
        )
    }
}

